api/src/test/java/org/pmiops/workbench/api/WorkspacesControllerTest.java

```diff
@@ -917,4 +924,35 @@ public void testUnableToShareWithNonExistentUser() throws Exception {
     shareWorkspaceRequest.addItemsItem(writer);
     workspacesController.shareWorkspace(workspace.getNamespace(), workspace.getName(), shareWorkspaceRequest);
   }
+
+  @Test
+  public void testNoteBookList() throws Exception {
+    org.pmiops.workbench.firecloud.model.WorkspaceResponse fcResponse =
+        new org.pmiops.workbench.firecloud.model.WorkspaceResponse();
+    org.pmiops.workbench.firecloud.model.Workspace mockWorkspace = new org.pmiops.workbench.firecloud.model.Workspace();
+    mockWorkspace.setBucketName("MockBucketName");
+    fcResponse.setWorkspace(mockWorkspace);
+    when(fireCloudService.getWorkspace("mockProjectName", "mockWorkspaceName")).thenReturn(
+        fcResponse
+    );
+    List<String> blobDetailList = new ArrayList<String>();
+    String mockBlobDetail1 = new String("notebook/File.ipynb");
```

Comment:
Why `new String()`? Can just put the string literal.
Also, why assign these to local vars at all?

`blobDetailList.add("...")`